Code/Framework/AzNetworking/AzNetworking/Serialization/AzContainerSerializers.h

```diff
@@ -321,4 +321,19 @@ namespace AzNetworking
             return serializer.IsValid();
         }
     };
+
+    template <>
+    struct SerializeObjectHelper<AZ::Aabb>
+    {
+        static bool SerializeObject(ISerializer& serializer, AZ::Aabb& value)
+        {
+            AZ::Vector3 minValue = value.GetMin();
+            AZ::Vector3 maxValue = value.GetMax();
+            serializer.Serialize(minValue, "minValue");
+            serializer.Serialize(maxValue, "maxValue");
```

Comment:
Could we delta serialize maxValue against minValue?

Reply:
It's tempting, but that's tricky to do for a generic type like this, since flt_max - (-flt_max) is not representable in a float, and any kind of float quantization results in a precision loss that we should preserve across both endpoints for simulation consistency. This is why QuantizedValues is structured in the way that it is. Delta packing structures works much better on structures that have known limits on numerical ranges that we can take advantage of.